modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -3107,6 +3107,25 @@ _objectDefinition1, _addObjectEntry(_objectDefinition1, null, 1),
 
 		AssertUtils.assertEquals(
 			WorkflowConstants.STATUS_DRAFT, status.getCode());
+
+		// Status Expired
+
+		_objectDefinition4.setEnableObjectEntryDraft(false);
+
```

Comment:
Can you call `objectDefinitionLocalService.updateObjectDefinition(` here? Like we have above after changing `enableObjectEntryDraft` property.